src/types/onyx/Report.ts

```diff
@@ -94,6 +94,9 @@ type Report = {
     /** The report type */
     type?: string;
 
+    /** If the admin room should be opened */
+    openOnAdminRoom?: boolean;
+
```

Comment:
Why is this being added?

Reply:
Without it we will have next TS error in` ReportScreenIDSetter.ts` file
![image](https://github.com/Expensify/App/assets/23176449/0040451e-0650-4362-bdc4-cfd476c313ce)